### ISSUE TITLE
With 3.14.0, fails with: couldn't find the config "./configs/base" to extend from

### ISSUE THREAD

**Issue by mat813** [B2o5T/graphql-eslint#1343]
### Issue workflow progress

_Progress of the issue based on the
[Contributor Workflow](https://github.com/the-guild-org/Stack/blob/master/CONTRIBUTING.md#a-typical-contributor-workflow)_

- [ ] 1. The issue provides a reproduction available on GitHub, Stackblitz or CodeSandbox
     > Please make sure the graphql-eslint version under `package.json` matches yours.
- [ ] 2. A failing test has been provided
- [ ] 3. A local solution has been provided
- [ ] 4. A pull request is pending review

---

**Describe the bug**

After updating to 3.14.0, I get:

```
$ eslint --fix --cache .

Oops! Something went wrong! :(

ESLint: 8.30.0

ESLint couldn't find the config "./configs/base" to extend from. Please check that the name of the config is correct.

The config "./configs/base" was referenced from the config file in "/.../ap/node_modules/@graphql-eslint/eslint-plugin/cjs/configs/schema-recommended.js".

If you still have problems, please stop by https://eslint.org/chat/help to chat with the team.
```

**To Reproduce** Steps to reproduce the behavior:

My config has:

```json
     {
        "files": [
          "*.graphql"
        ],
        "extends": "plugin:@graphql-eslint/schema-all",
        "parserOptions": {
          "schema": "./graphql/schema.graphql"
        },
        "rules": {
          "prettier/prettier": "error",
     }
```

**Expected behavior**

To not crash?

**Environment:**

- OS:Arch Linux
- `@graphql-eslint/eslint-plugin`: 3.14.0
- Node.js:v19.3.0

**Additional context**

If I edit `node_modules/@graphql-eslint/eslint-plugin/cjs/configs/schema-recommended.js` and replace `extends: './config/base'` with `extends: './base'` it works again.

**Comment by B2o5T** [comment]
> If I edit node_modules/@graphql-eslint/eslint-plugin/cjs/configs/schema-recommended.js and replace extends: './config/base' with extends: './base' it works again.

but after this change when you extend `schema-recommended` config you'll get `ESLint couldn't find the config "./base" to extend from` error

**Comment by mat813** [comment]
> > If I edit node_modules/@graphql-eslint/eslint-plugin/cjs/configs/schema-recommended.js and replace extends: './config/base' with extends: './base' it works again.
> 
> but after this change when you extend `schema-recommended` config you'll get `ESLint couldn't find the config "./base" to extend from` error

Yes, I know sorry, I should have said, it's not the solution, I don't understand eslint enough to get what should be done.

**Comment by B2o5T** [comment]
don't worry :) anyway, huge thanks for the report! fixes will be released in a few minutes

**Comment by B2o5T** [comment]
released in `3.14.1`
